binding-mri/binding-util.h

```diff
@@ -227,8 +227,12 @@ rb_float_arg(VALUE arg, double *out, int argPos = 0)
 		*out = FIX2INT(arg);
 		break;
 
+        case RUBY_T_NIL :
+                *out = 0;
+                break;
+
```

Comment:
Wrong indentation (use tabs).

Reply:
Guess I have to configure my editor to not convert tabs ...

Reply:
Also would you happen to know what are the exact tab width etc settings for this project.